src/gcloud_authorized_user.rs

```diff
@@ -51,6 +51,10 @@ impl TokenProvider for GCloudAuthorizedUser {
         Ok(token)
     }
 
+    async fn email(&self) -> Result<String, Error> {
+        run(&["auth", "print-identity-token"])
```

Comment:
I guess this needs additional steps as it only returns the token but not the email.